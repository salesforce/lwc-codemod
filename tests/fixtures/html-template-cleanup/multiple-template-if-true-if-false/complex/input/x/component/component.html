<template>
    <template for:each={contacts} if:true={visible} if:false={visible}>
        <li key={contact.Id}>
            {contact.Name}, {contact.Title}
        </li>
    </template>
</template>