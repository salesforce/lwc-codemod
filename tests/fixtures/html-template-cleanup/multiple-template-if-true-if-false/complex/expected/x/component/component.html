<template>
    <template for:each={contacts} if:true={visible}>
        <li key={contact.Id}>
            {contact.Name}, {contact.Title}
        </li>
    </template>
</template>